profiler/test/Datadog.Profiler.IntegrationTests/Datadog.Profiler.IntegrationTests.csproj

```diff
@@ -37,13 +37,17 @@
     <PackageReference Include="FluentAssertions" Version="5.10.3" />
     <PackageReference Include="Google.Protobuf" Version="3.20.1" />
     <PackageReference Include="MessagePack" Version="1.9.11" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.9.4" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2" />
     <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
     <PackageReference Include="System.ValueTuple" Version="4.5.0" />
-    <PackageReference Include="xunit" Version="2.4.1" />
+    <PackageReference Include="xunit" Version="2.4.2" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3">
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
+    <PackageReference Include="DatadogTestLogger" Version="0.0.33">
```

Comment:
FYI, we use this instead: 

```xml
  <!-- This property ensures NuGet packages are copied to the bin folder. This happens automatically in netcoreapp3.0+ -->
  <PropertyGroup Condition=" $(DD_LOGGER_ENABLED) != 'false' AND '$(TargetFramework)' == 'netcoreapp2.1' ">
    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
  </PropertyGroup>

  <ItemGroup Condition=" $(DD_LOGGER_ENABLED) != 'false' ">
    <PackageReference Include="DatadogTestLogger" Version="0.0.33" ExcludeAssets="compile"/>
  </ItemGroup>
```

You may want to consider using `DD_LOGGER_ENABLED` too, so that it's not _always_ referenced/copied to the bin (e.g. for local development), otherwise you will likely get messed up history in ciapp

Reply:
will do